docs/console.rst

```diff
@@ -0,0 +1,181 @@
+Console
+=======
+
+.. autoclass:: ev3dev2.console.Console
+    :members:
+
+Examples:
+
+.. code-block:: py
+
+    #!/usr/bin/env micropython
+    from ev3dev2.console import Console
+
+    # create a Console instance, which uses the default font, clears the screen,
+    # and turns off the cursor
+    console = Console()
+
+    # display 'Hello World!' at row 5, column 1 in inverse, but clear the console first
+    console.text_at('Hello World!', row=5, column=1, clear_screen=True, inverse=True)
+
+.. code-block:: py
+
+    #!/usr/bin/env micropython
+    from time import sleep
+    from ev3dev2._platform.ev3 import INPUT_1, INPUT_2, INPUT_3
+    from ev3dev2.console import Console
+    from ev3dev2.sensor.lego import GyroSensor, ColorSensor
+
+    console = Console()
+    gyro = GyroSensor(INPUT_1)
+    gyro.mode = GyroSensor.MODE_GYRO_ANG
+    color_sensor_left = ColorSensor(INPUT_2)
+    color_sensor_right = ColorSensor(INPUT_3)
+
+    # show the gyro angle and reflected light intensity for both of our color sensors
+    while True:
+        angle = gyro.angle
+        left = color_sensor_left.reflected_light_intensity
+        right = color_sensor_right.reflected_light_intensity
+
+        # show angle in inverse color when pointing at 0
+        console.text_at("G: %03d" % (angle), column=5, row=1, clear_screen=True, text_color=(angle == 0))
```

Comment:
I like the selection of text_color based on some condition :)

Reply:
Thanks. Although you unwittingly surfaced some code fix-ups to clean up in that doc (the `text_color` arg is now `inverse`, and there's no `x` and `y`, its now `column` and `row`).